src/glossary.rst

```diff
@@ -664,13 +665,15 @@ Glossary
 
    start
    startup
-      A start is when the Cylc :term:`scheduler` runs a :term:`suite`
+      A start is when the Cylc :term:`scheduler` runs a :term:`workflow`
       for the first time. The scheduler is the program that
-      controls the suite and is what we refer to as "running".
+      controls the workflow and is what we refer to as "running".
 
-      A suite start can be either :term:`cold <cold start>` or
+      A workflow start can be either :term:`cold <cold start>` or
       :term:`warm <warm start>` (cold by default).
 
+      .. TODO Suites -> Workflows once metomi cheat-sheet is updated
+
```

Comment:
I expect we will remove that cheat sheet, can use the new one in the migration guide.